docs/default.nix

```diff
@@ -306,7 +306,7 @@ let
       remainingStr = builtins.elemAt groups 1;
       prevLen = builtins.stringLength prev;
       currLen = builtins.stringLength current;
-      longest = if currLen > prevLen then current else prev;
+      longest = lib.max currLen prevLen;
```

Comment:
This is incorrect; `longest` was a string, `max` returns a number

Reply:
Oh, now I see. Yes, my bad.